tasks/Minimum-Effort Tacit Coordination Game

```diff
@@ -0,0 +1,43 @@
+# Summary
+> Players are provided different equilibrium points that are pareto ranked and need to make a payoff decision that reaches a efficient equibilibrium point with uncertainty testing payoff dominance.  
+
```

Comment:
Summary- after reading the papers it makes sense but for a more comprehensible summary I was thinking it should say something like:
"Players within a group are provided with a set of numbers that they must choose from to make a payoff decision that reaches an efficient equilibrium point, with uncertainty testing payoff dominance. For example, if all players choose the highest value, the group reaches a pay-off dominant equilibrium, where players each receive the highest possible amount of money. If players choose different values, the group is in disequilibrium and they receive sub-optimal gains."

(also sorry for the duplicate comments I'm still figuring out github)